.up-env

```diff
@@ -4,5 +4,5 @@ export TYPE="channel"
 export CHANNEL="nightly"
 
 # Used when TYPE=source
-export ALGOD_URL="https://github.com/algorand/go-algorand"
-export ALGOD_BRANCH="master"
+export ALGOD_URL="https://github.com/jannotti/go-algorand"
+export ALGOD_BRANCH="avm-box"
```

Comment:
Currently points to the fork, but should revert this back before merging to master.